Fixes for Сartography Inaccuracies

ValianBlue opened the pull request:
Resolves #342.

_Locations in italics_ can be inserted into the 'Find Title' dialogue to find locations quicker.

<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Complete rework of the _Venomweb_ Vale in Tirisfal Glades.
- Added trees to the following:
  - Eastern Lordaeron (Western and Eastern Plaguelands)
  - Quel'thalas (Eversong Woods, Ghostlands, and Zul'Aman)
  - Ashenvale
  - Hyjal
  - Krarasang Wilds
  - _Paoquan_
- Trees removed from _Stoneplow_
- Reworked the _Blackwald_ region in Gilneas.
- _Gilneas_ City texture has been changed.
- The two rivers near _Sidosh_ have been redone.
- Azshara Fixes
  - The river west of _Ravencrest_ in Uldum has been removed.
  - The mountain range on the south edge of Azshara has been removed.
- Partial Darkshore Revamp
  - Rework of the Heightmap for the northern part of Darkshore. Includes a coastal shelf along with textured rock formations.
  - _Cliffspring_ River added.
- Added terrain for:
    - Plunder Isles and surrounding islands
    - Kul Tiras
    - Northrend
    - Zandalar
    - Broken Isles
    - Expedition Islands
    - Pandaria
- Removed trees in the water


<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Known Issues:
@zumbak04 
- [ ] The provinces for Ahn'Qiraj need to be redrawn. 
  - A placeholder (ID #6379) has been placed in the newly added lands to the southwest of the region.
- [ ] Replace locators
- [ ] Several provinces from the old implementation of Uldum are now in Qiraji lands and need to be converted.
- [ ] Filter counties with one barony

<details>
<summary>Heightmap Overlay & WoW Cata Map</summary>

![](https://cdn.discordapp.com/attachments/751109648869752954/892495119482236938/unknown.png)
</details>

<details>
<summary>Province View</summary>

![image](https://user-images.githubusercontent.com/40177753/135155460-7a470de3-076d-4d9f-a16a-cf598c21546b.png)
</details>

## Planned Changes & Requests
- Northrend requires its own overhaul. The revisions outlined in #342 for Northrend will be added onto a new Northrend-only rework issue.
- Pandaria overhaul.
- A new tree object with yellow leaves for Quel'thalas and Quel'danas. @Warcraft-GoA-Development-Team/3d-artists 
<details>
<summary></summary>

![](https://wow.zamimg.com/uploads/screenshots/normal/287247-explore-isle-of-queldanas.jpg)
</details>



<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
(This section shamelessly copied from @ercarp)

Consider the following when giving your feedback:

**Canonicity:** Verify the map changes are accurate to the information made available by official canon.

**Familiarity:** Observe whether or not the changes are recognizable and fitting for their respective locations.

**Aesthetics:** Comment on the general look and appeal on the changes. Suggestions should be as specific as possible.

Additionally, ensure that necessary gameplay features like Holdings are not obscured by map changes such as trees, water, or steep cliffs.

### -- @ValianBlue

usfshihy committed:
Durotar And Echo Isles Terrain Rework

zumbak04 committed:
Reverted Wrong Files

(GDC)

zumbak04 committed:
Merge pull request #360 from Warcraft-GoA-Development-Team/durotar-echo-isled-terrain

Durotar and Echo Isles Terrain

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

# Conflicts:
#	gfx/map/map_object_data/building_locators.txt

ValianBlue committed:
Issue #342 Fixes

- VB

ercarp reviewed (changes requested):
>Complete rework of the _Venomweb_ Vale in Tirisfal Glades.

Venomweb Vale looks much better now and a lot closer to how it's supposed to look in World of Warcraft. I even logged into Retail WoW for the first time in 4 months and flew around to make sure. 😅

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132606826-831dce0c-1bed-45f9-adae-e0ab4de41c82.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132606860-5ad462ab-2c56-434d-8ff6-9755da4ccf62.png)

</details>

>Added trees to the following:
>       -- Eastern Lordaeron (Western and Eastern Plaguelands)
>       -- Quel'thalas (Eversong Woods, Ghostlands, and Zul'Aman)
>       -- Ashenvale
>       -- Hyjal
>       -- Krarasang Wilds
>       -- _Paoquan_

### Eastern Lordaeron

Looks good. 👍

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132607083-3431b39d-5d52-48f7-a7b0-1a6644d94176.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132607071-e2b33dcb-cb36-4560-98d2-aa054556df36.png)

</details>

### Quel'Thalas

Looking good, but the tree density is incorrect as you said. However, the screenshot you provided directly from the Map Editor looks great. No idea what's causing it to show incorrectly in-game but I hope we can fix it.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132607309-e001b3e0-c6e3-47b3-a50f-59809c20c3e6.png)

**After**

_Map Editor_
![image](https://user-images.githubusercontent.com/78575425/132608550-2c573f6b-7aca-4b03-897f-48816565bb4c.png)

_In-Game_
![image](https://user-images.githubusercontent.com/78575425/132607280-82404cf3-dea0-443e-a747-59d149c6a505.png)

</details>

### Ashenvale

I'm curious as to why the western half of Ashenvale is using different trees? Shouldn't the whole area be filled with the kind of trees you added here? If not, please correct me.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132607495-8a607e59-e09c-4460-9294-6dc6293d2618.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132607477-ea9f0013-1e8f-4e8f-80af-54059bf79869.png)

</details>

### Hyjal

Looks good. 👍

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132607675-d92c43b4-8fdc-4888-97fa-e74e4fb48afc.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132607658-680e64e5-4422-443e-823c-629cb837df84.png)

</details>

### Krasarang Wilds

Krasarang Wilds looks much more like the lush jungle it should be. Great job. 👍

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132607814-c6aff3b5-aa2d-4dbc-a604-1d95ff8110d4.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132607770-cbdc532d-b384-496c-b408-5ec4ee5e1d6c.png)

</details>

### Paoquan

Much better now. I kinda wish the trees and terrain were a bit more blue-ish like they are in WoW, but it's not really a big deal (and besides, we would have to do this for Krasarang too if that were the case).

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132611756-046e6fe2-9995-4a73-8106-911a3e3d6279.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132611799-8e42be66-0800-4ca1-bf60-d2bedd035aad.png)

*WoW Screenshots*
![image](https://user-images.githubusercontent.com/78575425/132611620-884477ae-be91-4e51-a35a-c30a8cf44c2b.png)
![image](https://user-images.githubusercontent.com/78575425/132611701-7d19b292-bfae-41f4-b772-42b49d2cfd04.png)

</details>

>Trees removed from _Stoneplow_

There's a much more appropriate number of trees now compared to how that area is depicted in WoW. Great job. 👍

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132612135-22d6b167-6467-405e-b0da-d54ddd05b8e8.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132612114-e9ba32fd-298e-43aa-9632-b45c775c8eb4.png)

*WoW Screenshots*
![image](https://user-images.githubusercontent.com/78575425/132612058-6c69209c-a363-43a1-94d5-cfe11601044e.png)
![image](https://user-images.githubusercontent.com/78575425/132612071-7c81a4e3-715b-4b11-9c05-0044e782b5f0.png)
![image](https://user-images.githubusercontent.com/78575425/132612088-000b1358-b2db-456c-b586-deb98e9e0752.png)

</details>

>Reworked the _Blackwald_ region in Gilneas.

Blackwald looks much closer now to how it should. My only complaint would be the green grass around the holding, but I think that's something the game adds around every holding so it's not your mistake. I just wonder if there's a way to disable it for certain holdings.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132608942-daa0c7b7-4d83-45fe-b35e-809f03024076.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132608979-6f1d044a-97a8-4c49-b3f3-f77f32c364bd.png)

*WoW Screenshots*
![image](https://user-images.githubusercontent.com/78575425/132608857-138398c9-3ebc-4ca1-b60a-9a89b7ad6403.png)
![image](https://user-images.githubusercontent.com/78575425/132608886-4f7942cf-6262-43b9-b901-fa7cf8b37c5c.png)

</details>

> Gilneas City texture has been changed.

I assume this was changed to make space for a potential custom 3D asset? Either way, I like it a lot as I don't think terrain used by cities should be painted in using the Map Editor.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132609293-db5df891-ef60-4041-9431-1f379f63292c.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132609260-d9de2c1f-3ab1-43f6-9346-eb5fa96e11a7.png)

</details>

> The two rivers near _Sidosh_ have been redone.

I agree that these rivers should probably be thinner as they are also that way in WoW (relative to the rest of the river), but right now it looks a bit odd with the leftover terrain from the old river still there.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132612716-35de5e0c-788a-4e78-a006-3382e04df307.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132612736-d2695442-79cc-4858-b73d-fea955f68d00.png)

*WoW Screenshots*
![image](https://user-images.githubusercontent.com/78575425/132612694-dfade670-91e8-41b1-b550-1229fd2d8361.png)
![image](https://user-images.githubusercontent.com/78575425/132612704-f9abb16b-123a-4a69-9663-816891e7f37e.png)

</details>

>Azshara Fixes
>-- The river west of _Ravencrest_ in Azshara has been removed.
>-- The mountain range on the south edge of Azshara has been removed.

A much more faithful representation of pre-Cataclysm Azshara. Good job.

<details><summary>Screenshots</summary>

**Before**
![image](https://user-images.githubusercontent.com/78575425/132613065-f94c956a-eaaa-4660-99bd-9f92eb69a6fa.png)

**After**
![image](https://user-images.githubusercontent.com/78575425/132613050-304329dc-2cbd-4ed4-b3e2-10a3c65938b9.png)

*WoW Map*
![image](https://user-images.githubusercontent.com/78575425/132613028-dd3e27b5-8bbb-4f15-abd7-0aa288983669.png)

</details>

>Partial Darkshore Revamp
>-- Rework of the Heightmap for the northern part of Darkshore. Includes a coastal shelf along with textured rock formations.
>-- _Cliffspring_ River added.

Darkshore looks much better now. Good job. 👍

<details><summary>Click to expand</summary>

**After**
![image](https://user-images.githubusercontent.com/78575425/132613197-4ba2c8b3-9e7c-40e0-9192-f519643ccd17.png)

**Before**
![image](https://user-images.githubusercontent.com/78575425/132613226-e7c4991d-3add-4c3d-9978-3237696cef93.png)

</details>

>A new tree object with yellow leaves for Quel'thalas. @Warcraft-GoA-Development-Team/3d-artists

This is something I would also like to see added. It's got my vote. 👍

**Overall, everything here looks good to me for the most part. My only concerns were the following:**

- Half of Ashenvale uses different trees. Why?
- Something should be done about the river situation in Uldum. They should probably be added in `rivers.bmp` before merge. (This is the most important one and the reason I'm tagging this as "request changes".)
- Holding in Blackwald has green grass around it which kind of ruins the effect. If there's an easy way to disable this, we should do it. If not, don't worry about it for now.

ValianBlue committed:
@ercarp Requested Changes for PR #662

- VB

ValianBlue commented:
### 3ea1957 Update
@ercarp 

> Looking good, but the tree density is incorrect as you said. However, the screenshot you provided directly from the Map Editor looks great. No idea what's causing it to show incorrectly in-game but I hope we can fix it.

_I will leave it as it is in case we find a solution. If not, the trees can replaced with a different type._

> I assume this was changed to make space for a potential custom 3D asset? Either way, I like it a lot as I don't think terrain used by cities should be painted in using the Map Editor.

_This may come in the future._

>I'm curious as to why the western half of Ashenvale is using different trees? Shouldn't the whole area be filled with the kind of trees you added here? If not, please correct me.

>Half of Ashenvale uses different trees. Why?

_Oversight on my part. Western Ashenvale (excluding the Zoram Strand) now matches the rest of the zone._
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/132647202-ac813457-9afa-4834-b86c-3fbc551a7c0a.png)
</details>

>Much better now. I kinda wish the trees and terrain were a bit more blue-ish like they are in WoW, but it's not really a big deal (and besides, we would have to do this for Krasarang too if that were the case).

_Terrain color can be changed, tree colors are a task for the 3D Design Team. If needed, I can change the terrain in a Pandaria specific overhaul._

>I agree that these rivers should probably be thinner as they are also that way in WoW (relative to the rest of the river), but right now it looks a bit odd with the leftover terrain from the old river still there.

>Something should be done about the river situation in Uldum. They should probably be added in rivers.bmp before merge. (This is the most important one and the reason I'm tagging this as "request changes".)

_Looks like the rivers file did not save those changes. I added in a Boulders European map texture to represent the canals._

<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/132649024-e7ecd49e-c9a9-4b5b-8379-39b09916fd53.png)
</details>

>Holding in Blackwald has green grass around it which kind of ruins the effect. If there's an easy way to disable this, we should do it. If not, don't worry about it for now.

_I added dense pine trees to the Blackwald (which it should have had in the beginning, admittedly). While it's a bandaid fix, it does help reduce the glow somewhat. It seems to be an inherent thing in the holding objects, and I do not have the capabilities to fix it entirely._

<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/132646673-faa77e71-a300-4ce3-b8bc-93dfc1b80bcc.png)
</details>

ercarp reviewed (approved):
This addresses everything I wanted changed, so I'm pretty happy with everything now. Approved! ✔️

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 commented on gfx/map/map_object_data/generated/steppe_bush_01_generator.txt:
```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
```
> The tree density is incorrect for Quel'thalas, despite the map editor showing otherwise.

I don't know why it happens but it's for good.
The FPS drops when you look at Ashenvale.

I'll try to find why, but tree density in Ashenvale should be decreased.

ValianBlue replied:
_I certainly was overzealous with tree placement. I'll reduce it to 40-70% depending on how it looks in the next commit._

zumbak04 replied:
> _I certainly was overzealous with tree placement. I'll reduce it to 40-70% depending on how it looks in the next commit._

So, the reason may be `tree_leaf_01_mask` and `tree_leaf_02_single_mask`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132696503-e3a8ca1a-f828-4ede-a744-e8d86be0a6d6.png)
![image](https://user-images.githubusercontent.com/46384992/132696513-f3ab135a-1635-42c1-b8a5-d0f5bdb884ec.png)

</details>

Ashenvale uses the single mask and prob that's why it's so laggy. If it's true, I would stop using this mask at all.

zumbak04 replied:
My FPS doesn't drop but keep in mind trees may affect the game a lot.

zumbak04 commented on gfx/map/map_object_data/generated/steppe_bush_01_generator.txt:
```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
 	layer="grass_layer"
```
> Provincial lines are incorrect for Darkshore.

And for Azshara.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132693780-f859ae69-3402-4d1a-a02e-b4b611fe9b85.png)
![image](https://user-images.githubusercontent.com/46384992/132693797-fcfdcd7b-9354-4cea-8cdc-e6e9c5dfa68f.png)

</details>

What do you want to change to fix it? Height and texture map or province map?

ValianBlue replied:
_Provincial map. Azshara I am planning to upgrade in a separate issue but I will note that in this PR under Known Issues._

zumbak04 replied:
> _Provincial map. Azshara I am planning to upgrade in a separate issue but I will note that in this PR under Known Issues._

On what you based height map?

zumbak04 replied:
So I would move the Darkshore coastline back, changing shape if you want.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132877806-f4c814d2-c227-4bb5-8647-1697d1011b58.png)

</details>

Our Kalimdor is wider and it breaks proportions.

zumbak04 commented on gfx/map/map_object_data/generated/steppe_bush_01_generator.txt:
```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
 	layer="grass_layer"
 	pdxmesh="steppe_bush_01_mesh"
```
The shore looks kinda flat. You may notice there were bitches and cliffs on the WoW map.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132693960-c98a98f6-51a6-46f9-9df6-0b0816f5fb36.png)
![image](https://user-images.githubusercontent.com/46384992/132694347-fc4ba9ac-26bb-4be0-b08c-9546166a3c99.png)

</details>

ValianBlue replied:
_Aye. The current edits to Azshara are more meant to satisfy the inaccuracies listed in #342. As mentioned, I would like to do a full revamp as part of a separate issue in the future._

zumbak04 commented on gfx/map/map_object_data/generated/steppe_bush_01_generator.txt:
```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
```
Blackwald doesn't blend well with the rest of Gilneas.
It looks like a black hole.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132694628-64b96e7a-b402-40a7-9e00-91908960c2c3.png)

</details>

ValianBlue replied:
_I'll reduce the intensity on the colormap in the next commit. Perhaps in the future when Gilneas gets a darker coloration (I certainly think it needs one!), the Blackwald can be made darker again._

zumbak04 replied:
Should be even more blended. If you look at this place, you may notice red isn't a dominant color here. It's just the color of grass here and there with gray veins of the default Gilneas color.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132878494-ac0f98bb-c6f9-4237-8adc-f299d7ae2501.png)

</details>

zumbak04 replied:
By the way, if you need it, you can resize colormap.

zumbak04 replied:
Like, make it 2 times bigger.

ValianBlue committed:
@zumbak04 Requested Changes for PR #662

ValianBlue commented:
## ea131d3 Changelog
@zumbak04 
### Ashenvale
- Reduced tree density by 70%.
<details><summary></summary>

![image](https://user-images.githubusercontent.com/40177753/132843089-9b12e3bd-35b2-4a40-8106-cfefe5afcb18.png)</details>

_If there is still a significant decrease in framerate, I will redo Ashenvale with_ `tree_leaf_01_mask`. 

### Gilneas
- Replace colormap profile for the _Blackwald_ region.
<details><summary></summary>

![image](https://user-images.githubusercontent.com/40177753/132842974-5c8ef63f-d83a-4757-8f89-25b32616e987.png)</details>

_If the Blackwald is still too dark, I will save the colormap for future use and remove the Blackwald's coloration for the current PR._

zumbak04 commented:
@ValianBlue answer inside threads, not just on the PR.

ValianBlue committed:
@zumbak04 The Reshaping of Kalimdor

ValianBlue commented:
##  c5f1555 Changelog
@zumbak04 
### Kalimdor
- Heightmap reshaped to have greater representative accuracy.

_Western Kalimdor and Azshara saw the most changes. Several islands were adjusted including Feathermoon.
Islands like Sardor Isle, the Isle of Dread, Echo Isles, and the South Sea Islands were revised._
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/133051960-e3f6fae7-c65a-4c85-8c3a-376735cc12aa.png)
</details>
<hr>

_There are inaccuracies for elevation, impassable terrain, various smaller islands (especially in Azshara), among other things. This commit is to fix the continent-wide shape and scale of Kalimdor._

ValianBlue committed:
Shifted water pool to fit correctly the Maelstrom

ValianBlue committed:
Many PvPers don't like the Seething Shore, but it's back.

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 committed:
Update provinces.xcf

(GDC)

zumbak04 commented on gfx/map/map_object_data/building_locators.txt:
```diff
@@ -1759,7 +1759,7 @@
 		}
 		{
 			id=294
```
If you change the heightmap don't leave it flat. Make it pretty.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133262300-44796e5b-9c48-48d3-8433-82fc708b0b2f.png)

</details>

ValianBlue replied:
Fixed in 71bfe27. Pending review.

zumbak04 commented on gfx/map/map_object_data/building_locators.txt:
```diff
@@ -1759,7 +1759,7 @@
 			id=294
-			position={ 3673.000000 0.000000 2232.000000 }
+			position={ 3671.783203 0.000000 2236.604004 }
 			rotation={ -0.000000 -0.000000 -0.000000 1.000000 }
```
Don't forget to change provinces. May ask @ElMariuso for help since he may be interested.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133263037-52d38340-9fec-4faf-939b-4319f1202791.png)

</details>

ValianBlue replied:
Fixed in 3ca060b.

zumbak04 commented on gfx/map/map_object_data/building_locators.txt:
```diff
@@ -1759,7 +1759,7 @@
-			position={ 3673.000000 0.000000 2232.000000 }
+			position={ 3671.783203 0.000000 2236.604004 }
 			rotation={ -0.000000 -0.000000 -0.000000 1.000000 }
 			scale={ 1.000000 1.000000 1.000000 }
```
This island in Darkshore is gone.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133265288-98de318b-7178-4380-9126-8348273c638f.png)

</details>

ValianBlue committed:
Ahn'Qiraj Fix

ValianBlue committed:
Darkshore Island Fix

ValianBlue committed:
Colormap.dds Upscale

ValianBlue committed:
The Reshaping of Kalimdor: Provinces

ValianBlue committed:
Lots of fixes

ValianBlue committed:
South Seas bugfix

ValianBlue committed:
Uldum Rivers & Tel'Abim Heightmap fixes

ValianBlue committed:
The Great Ahn'Qiraji High Detail Rework (HDR)

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 reviewed (changes requested):
Overall, good job on correcting the map. I'm impressed!

I may visit the branch to add new provinces in Ahn'qiraj if you want.

ValianBlue committed:
Southwest Kalmidor Province Fixes Pt. 1

ValianBlue commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
```
Resolved in 5c6e0f5.

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
```
So I would blur coastline, it's too sharp.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135163232-2ab18f63-3677-465d-989c-66b2b9ecd0ad.png)

</details>

ValianBlue replied:
Resolved in e0982e4.

zumbak04 commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
```
I would make these isles pretty too.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

ValianBlue replied:
Resolved in e0982e4.

zumbak04 committed:
Corrected Province Borders

zumbak04 commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
 				}
 				b_ornagh = {
```
Fixed in d2895f84aed11751ed806429a0d7baadb4222b4a.

ValianBlue committed:
@zumbak04 Requested Changes

ValianBlue committed:
Heightmap Hotfix

ValianBlue committed:
Cartography bugfixes for issues #626, #625, #523, and #614. Need @zumbak04 to check the unit counter displacement visual bug associated with #626 and #625.

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 committed:
c_south_slough Cleared

Cleared mentioning of removed c_south_slough.
(GDC)

zumbak04 commented:
You forgot to clear all mentions of removed `c_south_slough`. Fixed in df0559cce1a0b2977189d5ce8855f74c0410edf3.

ValianBlue committed:
Province Corrections

ValianBlue committed:
Province Corrections GDC

ValianBlue committed:
New Fixes for Issues #625, and #626.

ValianBlue committed:
New Fixes for Issues #625, and #626. GDC.

ValianBlue commented:
# Changelog
@zumbak04 

Resolves #614 #623 #625 #626

## b2f9452;
>  Cartography bugfixes for issues #626, #625, #523, and #614. Need @zumbak04 to check the unit counter displacement visual bug associated with #626 and #625.

_*Note: Issue #523 was linked by mistake, the actual issue is #623._

### 614
- Several changes were made. First, a new province with ID `6380` was created specifically for the island that the "Temple of Atal'hakkar" resides on. It is now the capital holding of the county, and has the Temple holding type.
- The original "Atal'hakkar" province, ID `81`, was renamed to "Bloodmire," and converted to a Tribal holding type.
- The "North Slough" and "South Slough" provinces were switched to correct their locations.
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143159705-b086385d-b60c-456a-b422-802124ccc6ba.png)
</details>

### 623
- Resolved this by adding two straights to the "Sethrak Isle" that connect to provinces `2273` "Scalefang," and `6140` "Shibre."
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143159509-98bc242d-9288-404e-9164-413f3b0e57be.png)
</details>

- Adding another straight between provinces `2284` "Zulkraa" and `6159` "Ronjaty."
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143159571-55f3bd52-a7a8-4b55-9167-817b8c354cb6.png)
</details>

### 625, 626
- The naval locators were manually replaced, however, this led to an issue with the unit counters not changing location to match the actual unit location.

<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143159309-d534a391-3047-45d7-9cc1-69c78c0cc2a4.png)
</details>

## 1457d2b, b275417;
- Corrections were made using the mapFiller tool. Unused provinces were removed, and new ones redefined.


## edc331c, 29d6a64;
- A series of new ocean provinces were created to resolve the known issue observed in b2f9452. One in the "WESTERSEA:" `6382`. Three in the "SOUTH SEA:" `6384`, `6385`, and `6386`.
- Other existing ocean provinces were redrawn to move their center positions away further from land.
- The placeholder province added when Ahn'Qiraj was revamped has been redefined as `6383` "PLACEHOLDER AQ."
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143160283-0a5a28f2-0e8f-43cd-b5b1-2a05955b3a07.png)
</details>
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143160349-705acddf-1019-420a-b294-c6b35b6bed46.png)
</details>
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/40177753/143160390-7d16bdaa-62a4-48a8-8c58-bba2aba84a9b.png)
</details>

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -1,4 +1,4 @@
-﻿###### AUTO GENERATED CONTENT (c) CKTools2 0.1.0A ######
```
Province borders differ from geography here and there.
Note: couldn't screenshot all, there are more.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/143526880-7bd38ee0-9bb9-49a1-9923-d0b95ffb8a09.png)
![image](https://user-images.githubusercontent.com/46384992/143526948-33510da6-52ea-4753-88a7-f8bd173f74cf.png)
![image](https://user-images.githubusercontent.com/46384992/143526961-a09864e9-f02d-4cd1-9d02-4734f44e00e4.png)
![image](https://user-images.githubusercontent.com/46384992/143526970-d09f6a59-94b9-483d-9ccd-fbc854604435.png)
![image](https://user-images.githubusercontent.com/46384992/143527219-1f258412-78e2-4790-940e-c1da954c6029.png)

</details>

ValianBlue replied:
Noted, will resolve during the full Kalimdor revamp.

zumbak04 commented on common/landed_titles/01_landed_titles.txt:
```diff
@@ -1,4 +1,4 @@
-﻿###### AUTO GENERATED CONTENT (c) CKTools2 0.1.0A ######
+﻿###### AUTO GENERATED CONTENT (c) CKTools3 1.3.1A ######
 
```
This province is too tiny. It should be either expanded or merged with another one (but don't forget to clean up after)
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/143527069-063a374c-367d-420b-9fb8-18692d8813fd.png)

</details>

ValianBlue replied:
Noted, will resolve during the full Kalimdor revamp.

ValianBlue committed:
Map reference file for @zumbak04

ValianBlue committed:
TRANSLATE Project

ValianBlue committed:
TRANSLATE Project Heightmap+Provinces+Rivers

ValianBlue committed:
TRANSLATE Project Locators

ValianBlue committed:
TRANSLATE Project Provinces Fix

ValianBlue committed:
TRANSLATE Project Provinces Fix GDC

ValianBlue commented:
# TRANSLATE Project Changelog
@zumbak04 

## Commits 6252e2a, 8502d9a, 5f5eb2d 
- Azeroth as a whole has been changed. Every continent and island aside from Kalimdor and its outlying isles were translated to more accurate locations as defined by the Chronicles.
- All locators (excluding the exceptions noted above) have been randomly reassigned to fit the new translated locations. All provinces with errors that could be fixed have been, any remaining errors must be solved manually in future commits during terrain reworks.
- Heightmap, Provinces, and Rivers map files all changed.
- A multitude of ocean provinces were removed. Some of the tags were recycled for use in the creation of some new ocean provinces, most notably around Teldrassil, Bloodmyst, and Azuremyst isles. 

_Note: I will redraw most, if not all, ocean provinces as I revamp the terrain. The three islands mentioned above were changed as they are now farther apart from each other and Kalimdor, necessitating new provinces._

**HEIGHTMAP**
<details>
<summary>Before</summary>

![image](https://user-images.githubusercontent.com/40177753/144338623-3b22895b-f093-43c8-b27d-61308ff7a2c3.png)
</details>
<details>
<summary>After</summary>

![image](https://user-images.githubusercontent.com/40177753/144338649-ccb97aa4-a314-49bc-9423-42b90a43d2a6.png)
</details>

**PROVINCES**
<details>
<summary>Before</summary>

![image](https://user-images.githubusercontent.com/40177753/144338721-858993a8-d81d-4213-9811-94f308215ca9.png)
</details>
<details>
<summary>After</summary>

![image](https://user-images.githubusercontent.com/40177753/144338740-ae061753-668c-4d69-abd4-2eab8ef6a156.png)
</details>


**RIVERS**
<details>
<summary>Before</summary>

![image](https://user-images.githubusercontent.com/40177753/144338670-5f31db4f-a106-4935-b140-9161168ba70b.png)
</details>
<details>
<summary>After</summary>

![image](https://user-images.githubusercontent.com/40177753/144338680-a25cc905-6b56-47b6-869b-f90338b6bb61.png)
</details>


## Commits d860c3b, d6d0269 
- provinceDef.xls updated.
- The now unused ocean provinces have been removed from necessary cartography files.
- Some tags were recycled, a list of unused province tags is provided below:

```
Gap: 1497
Gap: 1498
Gap: 1502
Gap: 1504
Gap: 1578
Gap: 1626
Gap: 1864
Gap: 1866
Gap: 1876
Gap: 1877
Gap: 1878
Gap: 2029
Gap: 2093
Gap: 2094
Gap: 2096
Gap: 2104
Gap: 2107
Gap: 2111
Gap: 2287
Gap: 2301
Gap: 2325
Gap: 5050
Gap: 5070
Gap: 5072
Gap: 5633
Gap: 5634
Gap: 5637
Gap: 6378
Gap: 6379

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

zumbak04 commented on map_data/definition.csv:
```diff
@@ -603,7 +603,7 @@
 602;25;163;133;SOUTH SEA;x;
```
You need to move terrain maps, etc. from `Warcraft-Guardians-of-Azeroth-2\gfx\map\terrain` and `watercolor_rgb_waterspec_a.dds` from `Warcraft-Guardians-of-Azeroth-2\gfx\map\water`.

Also, tree masks from `Warcraft-Guardians-of-Azeroth-2\content_source\map_objects\masks`.

zumbak04 commented on common/province_terrain/00_province_terrain.txt:
```diff
@@ -1032,7 +1032,6 @@
 1575=desert
```
So these errors appeared, but maybe I'll fix them.
```
[18:57:46][map.cpp:949]: Province 1497 RESERVED1497 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1498 RESERVED1498 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1502 RESERVED1502 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1504 RESERVED1504 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1578 RESERVED1578 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1626 RESERVED1626 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1864 RESERVED1864 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1866 RESERVED1866 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1876 RESERVED1876 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1877 RESERVED1877 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1878 RESERVED1878 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2029 RESERVED2029 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2093 RESERVED2093 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2094 RESERVED2094 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2096 RESERVED2096 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2104 RESERVED2104 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2107 RESERVED2107 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2111 RESERVED2111 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2287 RESERVED2287 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2301 RESERVED2301 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2325 RESERVED2325 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5050 RESERVED5050 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5070 RESERVED5070 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5072 RESERVED5072 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5633 RESERVED5633 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5634 RESERVED5634 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5637 RESERVED5637 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 6378 RESERVED6378 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 6379 RESERVED6379 has no pixels in provinces.bmp
```

ValianBlue replied:
_I'll likely use those ID's to fill in new provinces._ 👍

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

misterCocopop committed:
Changes to howling fjord

Changes to howling fjord

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/master' into terrain-2

# Conflicts:
#	gfx/map/terrain/materials.settings

zumbak04 committed:
Revert Unwated Changes

(GDC)

misterCocopop committed:
Changes to Howling Fjord

Added icebergs, polished southern icy areas and worked on the western isles

misterCocopop committed:
Added rivers & lakes to howling fjord

ValianBlue committed:
Major WIP @zumbak04

ValianBlue committed:
The Legend of the WIP continues

Gaben1025 committed:
Terrain/Height for Plunder and surrounding isles

- Added Terrain and changed the height of some areas for the Plunder Isles and some isles surrounding this kingdom

Gaben1025 committed:
Minor corrections to ddda2d5

- Added some more terrain were there was gray
- Fixed a city falling off a small island

Gaben1025 committed:
Zanadalar, South Islands, Booty Bay

- Added Terrain/ Objects for Zandalar, Booty Bay and South Islands
- Removed palm trees inside the ocean

Gaben1025 committed:
Coldarra, Strand of Ancients, Removed Objects

- Added terrain for coldarra and surrounding areas
- Added terrain for strand of ancients
- Removed some trees that are too dense

Gaben1025 committed:
Borean Tundra/ Coldarra Touch up

- Added terrain and objects to borean tundra, changed some height for Geyser Fields
- Added the land scaring for Coldarra as well as objects

Gaben1025 committed:
Dragonblight

- Added terrain,height,objects for dragonblight

Gaben1025 committed:
Kul Tiras and Mechagon

- Added terrain and objects and heights for Mechagon and Kul Tiras

runtd committed:
Gilneas

- Added terrain and objects to gilneas

runtd committed:
Broken isles WIP

- started adding terrain

runtd committed:
Broken Isles

- added terrain & object for broken isles

Gaben1025 committed:
Redridge/Swamp of Sorrows/ Zul'Gurub/Blasted Lands

Gaben1025 committed:
Westfall

Gaben1025 committed:
Gnomeregan, Searing Gorge, Badlands, Burning Steppes

Gaben1025 committed:
Loch Modan/ Gnomeregon

- Added terrain and objects

Gaben1025 committed:
Wetlands terrain

Gaben1025 committed:
Elwynn Forest/Grim Batol/Stromgarde

Gaben1025 committed:
Grim Batol fix/ Hillsbrad/Arathi fix

Gaben1025 committed:
Touch ups/Hinterlands/Started Silverpine

Gaben1025 committed:
Silverpine/Tristfall/Removed uneeded trees

- Finished Terrain and objects for silverpine and tristfall
- Removed a lot of ocean trees
- Removed trees blocking city models

Gaben1025 committed:
Western Plaguelands/ Building vfx

- Finished Western Plaguelands terrain and objects
- Took some buildings out of the water

runtd committed:
Qule'thalas

updated terrain, height & objects for quel'thalas (eversong woods, quel'danas and tranquillien)

Gaben1025 committed:
Eastern Plaguelands/Central Islands/Kezan

- Added terrain and objects for eastern plaguelands
- Added terrain for heart of azeroth and other Islands near it
- Added terrain for kezan and surrounding areas

Gaben1025 committed:
Molten Clay/Turtle Rock/ Isle of Giants/Sholozar Basin

Gaben1025 committed:
Northrend mainland done/ Tree fishing

- Wintergrasp
- Sholozar finished
- Icecrown
- Stormy Peaks
- Zul'drak
- Grizzly Hills
- Howling Fjord
- Removed ocean trees

runtd committed:
Quel'thalas small fixes, pandaria wip

Gaben1025 committed:
Isle of Thunder/ Townlong Steepes

Gaben1025 committed:
Top half of pandaria done

Gaben1025 committed:
Molten Clay Fix/ Tel'abim

Gaben1025 committed:
Azuremyst/ Teldrassil

Gaben1025 committed:
Pandarian Tree Fishing, Removing trees hiding cities

runtd committed:
Eastern Kingdom polish

made some touchups to the eastern kingdom
- Made some zones more accurate to the in game map
- Made some terrain transition smoother
- Removed some trees in mountains

Gaben1025 committed:
Zandalar/Northrend touch up/ Pandaria/Azuremyst

- Touched up terrain in Northrend and Zandalar
- Finished Pandaria
- More tree fishing
- Moved cities around to look better

Gaben1025 committed:
Mulgore/Southern Barrens/ Dusthallow Marsh

Gaben1025 committed:
Desolace/Feralas/Barrens/ North Kalim Heightmap

ValianBlue committed:
Uldum Heightmap (95% complete)

Gaben1025 committed:
Hyjal/Winterspring

TheGroxEmpire reviewed (commented):
- There were quite few errors, bunch of them were also map.cpp error. I attached the error files below:

[error.log](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/9776230/error.log)

- The game is also now consuming more than 6.5 in the task manager:
<details>

![image](https://user-images.githubusercontent.com/47816228/195588346-86b21dc4-a6eb-41e0-a6b8-6e4b0a116421.png)
</details>

Gaben1025 committed:
Ashenvale/Darkshore

Gaben1025 committed:
Hyjal/Darkshore/Moonglade/Felwood

- Hyjal/Darkshore touch up
- Moonglade/Felwood finished

Gaben1025 committed:
Terrain/Height Azshara&Durotar

ValianBlue committed:
Durotar + Azshara + Mulgore coloring @Gaben1025

ValianBlue committed:
Merge remote-tracking branch 'origin/terrain-2' into terrain-2

ValianBlue committed:
Silithus Part 1

ValianBlue committed:
Mapping is Hard Sometimes..

ValianBlue committed:
Feralas Trees are hard to see through

ValianBlue committed:
Clear Mulgore colormap

ValianBlue committed:
S. Kali nearly complete
